cluster/gce/gci/configure.sh

```diff
@@ -656,4 +688,7 @@ ensure-container-runtime
 # binaries and kube-system manifests
 install-kube-binary-config
 
+# Log all metadata access not from approved processes.
+configure-gce-metadata-fw
```

Comment:
What do you think about moving this into configure-helper.sh?
Specifically, integrating this into the [configure-ip-firewall ](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L99)function.